run_simple_election.sh

```diff
@@ -0,0 +1 @@
+eg e2e --guardian-count=2 --quorum=2 --manifest=data/election_manifest_simple.json --ballots=data/plaintext_ballots_simple.json --spoil-id=25a7111b-4334-425a-87c1-f7a49f42b3a2
```

Comment:
A sample script that contains all the values for running e2e for a simple election.  Once we get to #574 this should probably go into docs.